src/Tribe/REST/V1/Endpoints/Attendee_Archive.php

```diff
@@ -81,9 +81,16 @@ public function get_documentation() {
 	 *
 	 * @param WP_REST_Request $request
 	 *
+	 * @since TBD Returns 401 Unauthorized if Event Tickets Plus is not loaded.
+	 *
 	 * @return WP_Error|WP_REST_Response An array containing the data on success or a WP_Error instance on failure.
 	 */
 	public function get( WP_REST_Request $request ) {
+		// Early bail: ET Plus must be active to use this endpoint.
+		if ( ! class_exists( "Tribe__Tickets_Plus__Main" ) ) {
```

Comment:
:no_entry_sign: String "Tribe__Tickets_Plus__Main" does not require double quotes; use single quotes instead

<sub>hash: 3eb3df06e5fd952bb1a55b069069f156</sub>